src/test/java/gomedic/commons/core/MessagesTest.java

```diff
@@ -0,0 +1,26 @@
+package gomedic.commons.core;
+
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+
+public class MessagesTest {
+    @Test
+    public void getSuggestions_misspelledCommand_correctSuggestionsReturned() {
+        String wrongCommand = "addle";
```

Comment:
Actually, whats the max distance that you have set for the command, seems that clear and addle is like 5 distance apart but seems still included here

Reply:
i dont have a max distance right now, but I am considering setting a threshold of maybe string.length / 2 to filter out the unwanted commands

Reply:
I think its possible